doc/apps/eval.rst

```diff
@@ -0,0 +1,70 @@
+.. _eval_command:
+
+********************************************************************************
+eval
+********************************************************************************
+
+The ``eval`` command is used to compare the ``Classification`` dimensions of two
+point clouds.
+::
+
+    $ pdal eval predicted truth --labels LABELS
```

Comment:
I think this would be better as:

`$ pdal eval <predicted> <truth> --labels <labels>`

Otherwise it looks like `predicted` and `truth` are literals.

Reply:
Agree. Push this doc edit and it's ready to merge.